spec/metanorma_spec.rb

```diff
@@ -148,10 +148,12 @@ def ruby(x)
   its(:stdout) { is_expected.to include "bogus_format format is not supported for this standard" }
 end
 
-RSpec.describe "warns when no file provided" do
-  command "metanorma -t iso -x html"
-  its(:stdout) { is_expected.to include "Need to specify a file to process" }
-end
+# Maybe use the thor default error in this case?"
+#
+# RSpec.describe "warns when no file provided" do
```

Comment:
Can we customize Thor's error message? We can use the Thor default for now.

Reply:
I will look into it, thanks :)